addons/hr_contract/static/tests/hr_contract_mock_models.js

```diff
@@ -0,0 +1,142 @@
+import { fields, models, defineModels } from "@web/../tests/web_test_helpers";
+import { Domain } from "@web/core/domain";
+import { hrModels } from "@hr/../tests/hr_test_helpers";
+
+export class HrContract extends models.Model {
+    _name = "hr.contract";
+
+    name = fields.Char();
+    state = fields.Selection({
+        selection: [
+            ["draft", "New"],
+            ["open", "Running"],
+            ["close", "Expired"],
+            ["cancel", "Cancelled"],
+        ],
+    });
+    kanban_state = fields.Selection({
+        selection: [
+            ["normal", "Grey"],
+            ["done", "Green"],
+            ["blocked", "Red"],
+        ],
+    });
+    employee_id = fields.Many2one({ relation: "hr.employee" });
+    date_start = fields.Date();
+    date_end = fields.Date();
```

Comment:
I think we could use `ServerModel`

```suggestion
export class HrContract extends models.ServerModel {
    _name = "hr.contract";
```